static/panes/compiler.ts

```diff
@@ -2826,7 +2826,11 @@ export class Compiler extends MonacoPane<monaco.editor.IStandaloneCodeEditor, Co
         this.gccDumpButton.toggle(!!this.compiler.supportsGccDump);
         this.gnatDebugTreeButton.toggle(!!this.compiler.supportsGnatDebugViews);
         this.gnatDebugButton.toggle(!!this.compiler.supportsGnatDebugViews);
-        this.executorButton.toggle(!!this.compiler.supportsExecute);
+        if (!this.compiler.supportsBinary && !this.compiler.supportsExecute) {
+            this.executorButton.toggle(false);
+        } else {
+            this.executorButton.toggle(true);
```

Comment:
```suggestion
this.executorButton.toggle(this.compiler.supportsBinary || this.compiler.supportsExecute);
```

I think maybe is clearer? So, my read is now that all this "supportExecute" is superceded by supporting binary too? If that's the case then maybe we should add a new compiler function that returns this; then we won't need to look at both these flags here and in the other lines that check this?

like: `this.executorButton.toggle(this.canExecute());` and then
```ts
canExecute(): bool { return this.compiler.supportsExecute || this.compiler.supportsBinary; }
```
type stuff?

More generally, and maybe I need to re-read your comment at the top of the PR; should be introduce a new concept that's like "noReallyCanExecuteAtLeastSomewhere" (obviously with a better name :))